pysat/instruments/icon_euv.py

```diff
@@ -61,31 +83,35 @@ def init(self):
         modified in-place, as desired.
 
     """
-    logger.info("Mission acknowledgements and data restrictions will be printed " +
-          "here when available.")
+    logger.info(mm_icon.ackn_str)
+    inst.meta.acknowledgements = mm_icon.ackn_str
+    inst.meta.references = ''.join((mm_icon.refs['mission'],
+                                    mm_icon.refs['euv']))
+    if 'keep_original_names' in inst.kwargs.keys():
+        inst.keep_original_names = inst.kwargs['keep_original_names']
+    else:
+        inst.keep_original_names = False
+
     pass
 
 
 def default(inst):
-    """Default routine to be applied when loading data.
+    """Default routine to be applied when loading data. Adjusts epoch timestamps
+    to datetimes and removes variable preambles.
 
     Parameters
     -----------
     inst : (pysat.Instrument)
         Instrument class object
 
-    Note
-    ----
-        Removes ICON preamble on variable names.
-
     """
 
-    import pysat.instruments.icon_ivm as icivm
-    inst.tag = 'level_2'
-    icivm.remove_icon_names(inst, target='ICON_L2_EUV_Daytime_OP_')
+    mm_gen.convert_timestamp_to_datetime(inst, sec_mult=1.0e-3)
+    if not inst.keep_original_names:
+        mm_gen.remove_leading_text(inst, target='ICON_L26_')
 
 
-def load(fnames, tag=None, sat_id=None):
+def load(fnames, tag=None, sat_id=None, keep_original_names=False):
```

Comment:
```suggestion
def load(fnames, tag=None, sat_id=None, keep_original_names=None):
```
Since the default value must be specified in the init routine, should we specify this as a None here?